client/src/components/widgets/MessageDisplay.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div
+    class="alert m-3"
```

Comment:
I think how a component is displayed should be left to the parent.

Reply:
Not sure what you mean by this. Could you elaborate?

Reply:
Adding a margin to a component container is not something you expect when you are adding a component in your view, etc.
Deciding how a compoment should be positioned in relative to other is best left to the view/component that import it.